CHANGELOG_PENDING.md

```diff
@@ -22,6 +22,9 @@
 
 ### Bug Fixes
 
+- Document Fn::ToBase64 in README.
+  [#245](https://github.com/pulumi/pulumi-yaml/pull/245)
```

Comment:
IMO you don't need to include this to README as this is not a product change, just apply the `no-changelog-required` label.